Source/WebCore/rendering/RenderElement.cpp

```diff
@@ -2247,4 +2261,13 @@ FloatRect RenderElement::referenceBoxRect(CSSBoxType boxType) const
     return { };
 }
 
+bool RenderElement::shouldSkipContent() const
+{
+    if (style().contentVisibility() == ContentVisibility::Visible)
+        return false;
+    if (style().contentVisibility() == ContentVisibility::Hidden)
+        return true;
+    return element() && !element()->isRelevantToUser();
+}
```

Comment:
I think a switch(style().contentVisibility()) style would make it more obvious that isRelevantToUser belongs to ContentVisibility::Auto

Reply:
Done.